Index: u-boot-2025.10/arch/arm/dts/sun8i-h3-nanopi-neo.dts
===================================================================
--- u-boot-2025.10.orig/arch/arm/dts/sun8i-h3-nanopi-neo.dts
+++ u-boot-2025.10/arch/arm/dts/sun8i-h3-nanopi-neo.dts
@@ -72,5 +72,5 @@
 };
 
 &usbphy {
-	usb0_id_det-gpios = <&pio 6 12 GPIO_ACTIVE_HIGH>; /* PG12 */
+	usb0_id_det-gpios = <&pio 0 7 GPIO_ACTIVE_HIGH>; /* PG12  6 12 chg. pa07*/
 };
